CMakeLists.txt

```diff
@@ -228,6 +227,7 @@ TARGET_LINK_LIBRARIES(gimagereader
     ${TESSERACT_LDFLAGS}
     ${gimagereader_LIBS}
     ${SANE_LDFLAGS}
+    -ldjvulibre
```

Comment:
djvulibre has a pkgconfig file, it would be better to get include dirs and ldflags from pkgconfig instead of hard-coding them.

Reply:
Yep, i agree.